spec/ui/items_details_spec.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+Capybara.app = Sinatra::Application
+
+describe 'items details', type: :feature do
+  def visit_items_details_page_in_form
```

Comment:
We could dry this up further using a shared context to fill in the form in a `before`, and use `let`s for the things that change.

Reply:
_but_ might decrease test comprehension